samples/core/getting-started/unit-testing-using-dotnet-test/README.md

```diff
@@ -1,42 +1,19 @@
-Unit Testing using dotnet test Sample
-================
+# Unit testing using dotnet test sample
 
-This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-dotnet-test)
-for creating applications with unit tests included. See that topic for detailed steps on the code
-for this sample.
+This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-dotnet-test) for creating applications with unit tests included. See that topic for detailed steps on the code for this sample.
 
-Key Features
-------------
+## Key features
 
-This sample demonstrates creating a library and writing effective unit tests that validate the features in that library. 
+This sample demonstrates creating a library and writing effective unit tests that validate the features in that library. The example provides a service that indicates whether a number is prime.
 
-The example provides a service that indicates whether a number is prime.
+## Restore and test
```

Comment:
How do you think I feel? ..... On the first pass, I gave the darn blasted thing **an entry point!** lol